### PR TITLE
Create svx-1.3.1.md

### PR DESCRIPTION
**Pull request by anastasiaUsik**

### CODE REVIEW

**Comment by derek-meeco** [comment]
Now version 1.3.2

> Team, SVX Platform 1.3.2 has been released. The only component which has changed is Portal, from version 2.21.0 to 2.22.0

File: releases/svx-1.3.1.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+This release introduces several enhancements and bug fixes related to SVX Portal and Meeco Wallet
+
+* **Enhancements**: Introduced a new approach for displaying encrypted data for organization administrators depending on the entered passphrase. Added a Toolbar with links to Documentation and HelpDesk. Updated the empty screens "Credentials" and "Requests" with animations for Meeco Wallet.
+* **Bug Fixes**: Resolved a bug causing a 500 error during the opening/managing of verification responses. Fixed the ability to open Tenant or Organisation right after their creation. Various bug fixes for the Wallet application mostly related to User Experience improvements.
+  
+## Enhancements
+
+### Portal
+
+* Created a new Toolbar.
```
Comment (javereec) [inline releases/svx-1.3.1.md]: @anastasiaUsik can you specify where you can find this Toolbar

File: releases/svx-1.3.1.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+* Updated the screen View credential based on security aspects and whether a passphrase was entered or not.
```
Comment (javereec) [inline releases/svx-1.3.1.md]: What was the security aspect that was enhanced?

**Review by javereec — COMMENTED** [review]
Do these release notes address 1.3.1 or 1.3.2.
Have a look here for paltform changelog: https://bitbucket.org/meeco/gateway/src/develop/PLATFORM.md

File: releases/svx-1.3.1.md (hunk 2 above)
Comment (anastasiaUsik) [reply releases/svx-1.3.1.md]: @javereec as far as I know Terry did some work, could you please help me with rewriting this point?
everything else has been updated

**Review by linasi — APPROVED** [review]

**Pull request merged**